src/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,836 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+** Class: UnsafeNativeMethods
+**
+============================================================*/
+namespace Microsoft.Win32
+{
+
+    using Microsoft.Win32;
+    using Microsoft.Win32.SafeHandles;
+    using System;
+    using System.Configuration.Assemblies;
+    using System.Runtime.CompilerServices;
+    using System.Runtime.ConstrainedExecution;
+    using System.Runtime.InteropServices;
+    using System.Security.Principal;
+    using System.Runtime.Serialization;
+    using System.Threading;
+    using System.Runtime.Versioning;
+    using System.Security;
+    using System.Security.Permissions;
+    using System.Text;
+    using System.Diagnostics.Eventing;
+    using System.Diagnostics.Eventing.Reader;
+
+    [SuppressUnmanagedCodeSecurityAttribute()]
+    internal static partial class UnsafeNativeMethods
```

Comment:
are we using all the constants and apis defined here? remove all the unneccessary constant and apis. Also check if they already exist or not. because some of them may already exist in the repo